.rhdh/docker/README.adoc

```diff
@@ -0,0 +1,15 @@
+# Downstream Dockerfile
+
+This folder contains the containerfile (dockerfile) used in Konflux downstream for RHDH builds.
+
+It requires hermeto to cache dependencies (rpm, pip, yarn) before the build can proceed, so that the build is reproduceable/offline/hermetically sealed.
+
+## How do I build this dockerfile
+
+Depending on what you're trying to accomplish you have some options:
+
+* build from the link:../../docker/Dockerfile[docker/Dockerfile] instead of link:Dockerfile[.rhdh/docker/Dockerfile], using `podman build` or equivalent
+
+* create your own hermeto cache folder with https://github.com/hermetoproject/hermeto and then tweak this Dockerfile using ENV vars, to use your local cache folder
```

Comment:
So I've done this, but it's not clear to me from [this commented out block](https://github.com/redhat-developer/rhdh/blob/main/.rhdh/docker/Dockerfile#L176-L181) how the cache is supposed to be made available to the build.  I opted to just for the short term hack away at the file and copy the cache folder somewhere etc. which did get me a bit further.

Reply:
gotcha. So we need to explain better that by setting

`YARN_GLOBAL_FOLDER=/cachi2/output/deps/yarn`

to point you your local `/path/to/hermeto-output/deps/yarn` folder instead

you should be able to have the build behave "as if offline" 

There's also https://github.com/hermetoproject/hermeto/blob/main/docs/usage.md#generate-environment-variables to explain how to generate the env vars.